packages/sanity/src/form/inputs/Slug/utils/slugify.ts

```diff
@@ -9,12 +9,16 @@ const defaultSlugify = (value: FIXME, type: SlugSchemaType): string => {
   return value ? speakingurl(value, slugifyOpts) : ''
 }
 
-export function slugify(sourceValue: FIXME, type: SlugSchemaType): Promise<string> {
+export function slugify(
```

Comment:
suggestion: same as above re: converting async function. I think we want the error to propagate on the returned promise (I do realise that this isn't "new" behavior, so feel free to ignore)